scripts/provisioning/s3_setup

```diff
@@ -247,58 +230,10 @@ then
 setup_release_repo
 fi
 
-# Install required S3 release rpm dependencies.
```

Comment:
removed this as this section seems to be obsolete. Need review from @shalakadharap , @AjinkyaDhumal , @sachitanands